zulip_bots/zulip_bots/bots/tictactoe/doc.md

```diff
@@ -1,28 +1,30 @@
-# About Tic-Tac-Toe Bot
+# Tic-Tac-Toe Bot
 
 This bot allows you to play tic-tac-toe in a private message with the bot.
 Multiple games can simultaneously be played by different users, each playing
 against the computer.
 
 The bot only responds to messages starting with @mention of the bot(botname).
 
-### Commands
-**@mention-botname new** will start a new game (but not if you are
+### Usage
+The Tic-Tac-Toe Bot has four commands:
+1. `@mention-botname new` - start a new game (but not if you are
 already playing a game.) You must type this first to start playing!
 
-**@mention-botname help** will return a help function with valid
+2. `@mention-botname help` - will return a help function with valid
 commands and coordinates.
 
-**@mention-botname quit** will quit from the current game.
+3. `@mention-botname quit` - quit from the current game.
 
-**@mention-botname <coordinate>** will make a move at the
-entered coordinate. For example, **@mention-botname 1,1** . After this, the bot will make
-its move, or declare the game over if the user or bot has won.
+4. `@mention-botname <coordinate>` - make a move at the entered coordinate.
```

Comment:
Enumerations make most sense when you want the reader to execute smt. in a specific order, or rank the listed items. In this case, bullet points seem to be more appropriate.